pydis_site/settings.py

```diff
@@ -48,10 +49,26 @@
     ALLOWED_HOSTS = env.list('ALLOWED_HOSTS', default=['*'])
     SECRET_KEY = "yellow polkadot bikini"  # noqa: S105
 
+    # Prevent verbose warnings emitted when passing a non-timezone aware
+    # datetime object to the database, whilst we have time zone support
+    # active. See the Django documentation for more details:
+    # https://docs.djangoproject.com/en/dev/topics/i18n/timezones/
+    warnings.filterwarnings(
+        'error', r"DateTimeField .* received a naive datetime",
+        RuntimeWarning, r'django\.db\.models\.fields',
+    )
```

Comment:
Wait wait wait, I don't get it. You're solving this problem so that we're passing timezone-aware objects, right? Why do we then still need to surpress the warnings?

Reply:
this is a win, and we are all part of it.

Reply:
nonononono, it will raise an _error_ because it got that runtime warning, that's why I pass `'error'` as the first argument. I actually stole this, I did not write it myself